nutkit/protocol/responses.py

```diff
@@ -81,9 +88,10 @@ class BookmarksSupplierRequest:
     bookmarks for all databases.
     """
 
-    def __init__(self, id, database=None):
+    def __init__(self, id, bookmarkManagerId, database=None):
         # id of the callback request
         self.id = id
+        self.bookmarkManagerId = bookmarkManagerId
```

Comment:
```suggestion
        self.bookmark_manager_id = bookmarkManagerId
```

I'm ok with breaking Python's style guide for the things that go over the wire. That's why the request classes have attributes that are camel case instead of snake case. Hence, I'm ok with the response classes having camel case parameters. But right after that one layer, I'm voting for re-writing it to be PEP8 conform.